kitsune/search/v2/es7_utils.py

```diff
@@ -103,7 +103,14 @@ def index_objects_bulk(doc_type_name, obj_ids):
     doc_type = next(cls for cls in get_doc_types() if cls.__name__ == doc_type_name)
 
     objects = doc_type.get_queryset().filter(pk__in=obj_ids)
+    # prepare the docs for indexing
     docs = [doc_type.prepare(obj).to_dict(include_meta=True) for obj in objects]
+    # Replace _source with doc in order to do a bulk update
+    for doc in docs:
+        doc["doc"] = doc["_source"]
+        del doc["_source"]
+        doc.update({"_op_type": "update", "doc_as_upsert": True})
```

Comment:
Same comment re updating here, and we should merge the list comprehension into the loop.